server/game/cards/06-WoE/HireOn.js

```diff
@@ -0,0 +1,30 @@
+const Card = require('../../Card.js');
+
+class HireOn extends Card {
+    // Play: Make a token creature. If there is a combined total of 6
+    // Aember or more between both players' pools, archive Hire On.
+    setupCardAbilities(ability) {
+        this.play({
+            gameAction: [
+                ability.actions.makeTokenCreature(),
+                ability.actions.conditional({
+                    condition: (context) =>
+                        context.player.amber + context.player.opponent.amber >= 6,
```

Comment:
Need to check (context.player.opponent ? context.player.opponent.amber : 0)

Reply:
Is that to handle one-player games?

Reply:
Yes